### DISCUSSION TITLE
How to use regularization images?

### DISCUSSION THREAD

**Discussion by TeKett** [bmaltais/kohya_ss#1485] [Q&A]
I added a folder with images, but it says "num reg images: 0" when i start, and it doubles the epochs.

**Comment by mistermcluvin** [comment]
you added the REG folder and under it you put the 1_man or 1_woman folder with your images? Where 1 is the number of times it will repeat the processing. Similar to your Images folders, do not pick the 1_man folder in the GUI, pick the parent REG folder and Kohya will find what is in it (repeats_class).
